main.tf

```diff
@@ -2,53 +2,44 @@ data "aws_caller_identity" "default" {}
 
 data "aws_region" "default" {}
 
-resource "aws_s3_bucket" "cache_bucket" {
+module "cache_bucket" {
```

Comment:
Usually when we create a breaking change like this, we release a "minor" release but we also include with it some kind of "migration" document which we create by first applying the current release, then running through terraform state moves and other API changes, then we can apply the new version (this version) and make sure it returns "no changes".

Those steps are documented inside of docs/ and added to the README.yaml so other people will know how to safely migrate from version 0.x to version 0.y.

Reply:
I ran the tests below just to make sure you were on the correct path with the current proposed changes but I think a migration doc is the right way to go.

cc: @Nuru

Reply:
I added a migration doc @nitrocode 
Can you please review?